obs/CMakeLists.txt

```diff
@@ -63,10 +59,15 @@ add_subdirectory("${PROJECT_TOP}/common"          "${CMAKE_BINARY_DIR}/common")
 add_subdirectory("${PROJECT_TOP}/repos/LGMP/lgmp" "${CMAKE_BINARY_DIR}/lgmp"  )
 
 add_library(looking-glass-obs SHARED ${SOURCES})
-target_link_libraries(looking-glass-obs
+target_compile_options(looking-glass-obs PRIVATE ${C_FLAGS})
+target_link_libraries(looking-glass-obs PRIVATE
 	${EXE_FLAGS}
 	lg_common
 	lgmp
 )
 
+install(TARGETS looking-glass-obs
```

Comment:
what about a local install into the local obs plugins folder for the user?

Reply:
Good point, that might make sense as an option... `${CMAKE_INSTALL_DATADIR}/obs/obs-plugins/` becomes `$ENV{HOME}/.config/obs-studio/plugins/` I think?